site/src/components/button.tsx

```diff
@@ -80,10 +80,11 @@ export type ButtonProps = {
 // probably rename to BPButton?
 // @todo implement loading
```

Comment:
This wasn't introduced in this PR, but please can you help remove this. `loading` has been implemented here. The renaming comment is also no longer needed